tools/setup_discord.py

```
@@ -0,0 +1,414 @@
+        "Content-Type": "application/json",
+        "User-Agent": "LogisticsSetupScript/1.0",
+    }
+
+
+def api(method: str, path: str, token: str, body: Optional[dict] = None) -> dict:
+    """Make a Discord API call, respecting rate limits automatically."""
+    url = f"{BASE}{path}"
+    data = json.dumps(body).encode() if body is not None else None
+    req = urllib.request.Request(url, data=data, headers=_headers(token), method=method)
+
+    for attempt in range(5):
```

Ruff (0.15.12)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)